Podcast/ImageView.swift

```diff
@@ -12,6 +12,14 @@ class ImageView: UIImageView {
         }
     }
     
+    func setImageAsynchronouslyWithDefaultImage(url: URL?, defualtImage: UIImage = #imageLiteral(resourceName: "nullSeries")) {
```

Comment:
'defualt' lol
But more importanlty was there a reason to use this? You're not using `defualtImage` anywhere

I really am not sure what this class is for, especially since Haneke has the ability to set default image and completion callback and we aren't using either

Reply:
I do use the defaultImage override in a lot of places. I removed the completion callback